code/modules/surgery/organs_internal.dm

```diff
@@ -120,6 +120,8 @@
 					target.sdisabilities &= ~BLIND
 				if(I.organ_tag == O_LUNGS)
 					target.SetLosebreath(0)
+				if (I.status == ORGAN_DEAD)
```

Comment:
There already is a surgery that does this in other.dm. Why add two ways to do the same surgery?

Reply:
There isn't, actually. Both of the necrotic organ repair surgeries there only work on external organs, and reworking that to work on internal organs as well still wouldn't solve the 'you can fully repair an organ but still have it be an unkillable zombie organ' problem.

Reply:
Ah, I see. But then shouldn't you have changed line 114? Because you cannot even get to the part where it unsets dead if it has no damage. If you end up with an organ that has 0 damage somehow, but still has the dead flag, 114 will be false and you will not be able to fix it still. Shouldn't it maybe be `if(I && (I.damage > 0 || I.status & DEAD)`?

Reply:
Maybe, though the only ways that could happen is being injected with 60+ units of peridaxon (which overdoses at 10u) or through varediting the damage away (which can also be done to the dead status in that case).

Reply:
You saaaay that, and yet you're here fixing a problem that allows exactly that to happen. :p The ability to completely heal dead organs and not remove the dead flag. You might as well just make it future-proof.